src/west/commands/configuration.py

```diff
@@ -0,0 +1,88 @@
+# Copyright (c) 2019, Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: Apache-2.0
+
+'''West config commands'''
+
+import argparse
+import configparser
+from west import config
+from west.config import ConfigFile
+from west.commands import WestCommand
+
+CONFIG_DESCRIPTION = '''\
+West configuration file handling.
+
+Command to assist user in getting or setting west project setting
+'.west/config' or west user configuration file '~/.westconfig'.
+
+To get a value for <name>, type
+    west config <name>
+
+To set a value for <name>, type
+    west config <name> <value>
+'''
+
+
+class Configuration(WestCommand):
+    def __init__(self):
+        super().__init__(
+            'config',
+            'get or set configuration settings in west config files',
+            CONFIG_DESCRIPTION)
+
+    def do_add_parser(self, parser_adder):
+        parser = parser_adder.add_parser(
+            self.name,
+            help=self.help,
+            formatter_class=argparse.RawDescriptionHelpFormatter,
+            description=self.description)
+
+        parser.add_argument(
+            'name',
+            help='name is section and key seperated by a dot')
```

Comment:
*separated

Also, please add an example

Reply:
thanks for spotting.
Guess I didn't notice spell checker complaining :(

Will also add an example.